packages/config/src/projects/layer2s/cronoszkevm.ts

```diff
@@ -27,6 +111,362 @@ export const cronoszkevm: Layer2 = underReviewL2({
+      {
+        ...EXITS.REGULAR('zk', 'merkle proof'),
+        references: [
+          {
+            text: 'Withdrawing funds - ZKsync documentation',
+            href: 'https://docs.zksync.io/build/developer-reference/bridging-assets',
+          },
+        ],
+      },
+      EXITS.FORCED('forced-withdrawals'),
+    ],
   },
-  // shared escrow with zk stack
-})
+  contracts: {
+    addresses: [
+      discovery.getContractDetails('CronosZkEvm', {
+        description:
+          'The main Rollup contract. The operator commits blocks and provides a ZK proof which is validated by the Verifier contract \
+          then processes transactions. During batch execution it processes L1 --> L2 and L2 --> L1 transactions.',
+        ...upgrades,
+      }),
+      discovery.getContractDetails('Governance', {
+        description: `Intermediary governance contract with two roles and a customizable delay. 
+        This delay is only mandatory for transactions scheduled by the *Owner* role and can be set by the *SecurityCouncil* role. 
+        The *SecurityCouncil* role can execute arbitrary upgrade transactions immediately. 
+        Currently the delay is set to ${formatSeconds(
+          discovery.getContractValue<number>('Governance', 'minDelay'),
+        )}
+        ${isSCset ? '.' : ' and the *SecurityCouncil* role is not used.'}`,
+        ...upgrades,
+      }),
+      discovery.getContractDetails('CronosZkEVMAdmin', {
+        description:
+          'Intermediary governance contract that has the *Admin* (not upgradeability admin) role for the Cronos zkEVM diamond contract.',
+      }),
+      discovery.getContractDetails(
+        'ValidatorTimelock',
+        'Intermediary contract between the *Validators* and `CronosZkEvm` that delays block execution (ie. withdrawals and other L2 --> L1 messages).',
+      ),
+      discovery.getContractDetails('Verifier', {
+        description: 'Implements ZK proof verification logic.',
+        ...upgrades,
+        upgradeConsiderations:
+          'ML Multisig can change the verifier with no delay.',
+      }),
+      discovery.getContractDetails('L1SharedBridge', {
+        description:
+          'This bridge contract escrows all assets that are deposited to Cronos zkEVM and other chains in the Hyperchain Ecosystem.',
+        ...upgrades,
+      }),
+      discovery.getContractDetails('BridgeHub', {
+        description:
+          'Sits between the single shared bridge and the StateTransitionManager(s) and relays L1 <-> L2 messages from the shared bridge or other ZK stack chains to their respective destinations.',
+        ...upgrades,
+      }),
+      discovery.getContractDetails('StateTransitionManager', {
+        description:
+          'Defines shared L2 diamond contract creation and upgrade implementations and the verifier for the Hyperchain contracts connected to it.',
+        ...upgrades,
+      }),
+    ],
+    risks: [CONTRACTS.UPGRADE_NO_DELAY_RISK],
+  },
+  stateDerivation: {
```

Comment:
remove this section, we don't really know what's going on and it's written as if data is on L1

Reply:
okay, i had added it because they linked to the [zksync era state reconstructor](https://github.com/eqlabs/zksync-state-reconstruct?tab=readme-ov-file) on telegram, but agree